subsys/bluetooth/audio/ascs.c

```diff
@@ -51,17 +55,57 @@ LOG_MODULE_REGISTER(bt_ascs, CONFIG_BT_ASCS_LOG_LEVEL);
 struct bt_ascs_ase {
 	struct bt_ascs *ascs;
 	struct bt_audio_ep ep;
+	const struct bt_gatt_attr *attr;
 };
 
 struct bt_ascs {
 	struct bt_conn *conn;
-	struct bt_ascs_ase ases[ASE_COUNT];
+	struct bt_ascs_ase *ases[ASE_COUNT];
 };
 
+K_MEM_SLAB_DEFINE(ase_slab, sizeof(struct bt_ascs_ase),
+		  CONFIG_BT_ASCS_MAX_ACTIVE_ASES, __alignof__(struct bt_ascs_ase));
+
 static struct bt_ascs sessions[CONFIG_BT_MAX_CONN];
 
 static int control_point_notify(struct bt_conn *conn, const void *data, uint16_t len);
 
+static void bt_ascs_ase_set_inactive(struct bt_ascs_ase *ase)
+{
+	__ASSERT(ase && ase->ascs, "Non-existing ASE or ASCS");
+
+	for (uint8_t i = 0; i < ASE_COUNT; i++) {
```

Comment:
```suggestion
	for (int i = 0; i < ASE_COUNT; i++) {
```

`ASE_COUNT` is an int